jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -205,6 +205,19 @@ var module = module;
         // if the macaroon is valid.
         false
       );
+    },
+
+    /**
+      Lists all the templates for cloud credentials for a specific user.
```

Comment:
What user? We don;t pass a user here, so I guess it is "for the currently authenticated user"?